src/components/ArtistList.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+export default ArtistList
+
+ArtistList.propTypes = {
+  artists: PropTypes.array,
+}
+
+function ArtistList({ artists }) {
+  return (
+    <div>
+      <ListStyled>
+        {artists?.map(({ artist, songTitle, id }) => (
+          <ListItemStyled key={id}>
+            <span>{artist}</span>
+            {songTitle}
```

Comment:
Why did you enclose `{artist}` in a span but `{songTitle}` does not have a tag?

Reply:
`Artist` and `SongTitle` are both enclosed in the `list tag`. The `artist` gets an extra styling, which is why it is included in a `span tag`.